blockchain/accept.go

```diff
@@ -111,9 +111,7 @@ func (b *BlockChain) maybeAcceptBlock(block *btcutil.Block, flags BehaviorFlags)
 		// upgraded.  This is part of BIP0034.
 		if blockHeader.Version < 2 {
 			if b.isMajorityVersion(2, prevNode,
-				b.chainParams.BlockV1RejectNumRequired,
-				b.chainParams.BlockV1RejectNumToCheck) {
-
```

Comment:
Please don't nuke this extra line.  It's there for visual readability since the indentation is the same on the wrapped if.